### ISSUE TITLE
Typed texts show up only after pressing space with SwiftKey keyboard. 

### ISSUE THREAD

**Issue by aasutossh** [termux/termux-app#1473]
**Problem description**

As the title says. Some keyboards work though. 

**Steps to reproduce**

**Expected behavior**

**Additional information**

* Termux application version: 0.83
* Android OS version: 6.0
* Device model: Gionee F103 pro

**Comment by xeffyr** [comment]
Duplicate of https://github.com/termux/termux-app/issues/686.

> Android OS version: 6.0

Android OS versions **5.x** and **6.x** are not supported since 01.01.2020.

**Comment by aasutossh** [comment]
Okay cool.
Thank you.
